app.py

```diff
@@ -149,6 +153,35 @@ def get_book_info():
     )
 
 
+@app.route("/get_favorites", method="POST")
+def get_favorites():
+    favorites = Favorites.query.filter(Favorites.userEmail == current_user).all()
+    favorite_list = []
+    for i in favorites:
+        favorite_list.append(i.isbn)
+
+    # Need to make api calls for every isbn in the list and get new list based on that info and then pass that into html.
+
+
+# Need current user to store current user, so need to talk to maryam about user login
+# And make sure that what i am doing will work.
+@app.route("/add_favorite", method=["GET", "POST"])
+def add_favorite():
+    isbn = flask.request.form.get("isbn")
+    new_favorite = Favorites(username=current_user, bookISBN=isbn)
+    db.session.add(new_favorite)
+    db.session.commit()
+
+
+@app.route("/delete_favorite", method=["GET", "POST"])
```

Comment:
Keep this route as well. This will work after we add a form to delete a certain favorite ISBN of a book from the favorites table.